src/ceph_osd.cc

```diff
@@ -280,6 +280,27 @@ int main(int argc, const char **argv)
 	dout(5) << "object store type is " << store_type << dendl;
       }
       ::close(fd);
+    } else {
+      // hrm, infer the type
+      snprintf(fn, sizeof(fn), "%s/current", data_path.c_str());
```

Comment:
what we do in ceph-volume is check for the presence of `journal`. Is checking for the `current/` directory a better way?

Reply:
Both are required, so I think it doesn't matter.